packages/xpub-lib/src/derivation.js

```diff
@@ -73,7 +85,9 @@ function deriveAddress({ purpose, pubkey, network }) {
       return bc1qAddress
     }
     case Purpose.P2TR: {
-      // Context: https://bitcoinops.org/en/topics/x-only-public-keys/
+      if (!eccInstance) {
+        throw new Error("An instance of an ECC library implementing the secp256k1 curve must be initialized to generate taproot addresses. You must first call initEccLib().");
```

Comment:
Would be nice to add this requirement to the xpub-lib readme somewhere? Maybe as an added example?